perf.groovy

```diff
@@ -0,0 +1,32 @@
+// Import the utility functionality.
+
+import jobs.generation.*;
+
+def project = GithubProject
+def branch = GithubBranchName
+def projectFolder = Utilities.getFolderName(project) + '/' + Utilities.getFolderName(branch)
+
+
+[true, false].each { isPR ->
+    ['Windows_NT'].each { os ->
+        def newJob = job(Utilities.getFullJobName(project, "perf_${os}", isPR)) {
+            // Set the label.
+            steps {
+                    // Batch
+                    batchFile("set __TestIntermediateDir=int&&build.cmd release x64")
+                    batchFile("tests\\runtest.cmd release x64")
```

Comment:
That might work, as long as this step generates all files required for perf run, that should be good enough. Will make this change and give a test run.